### PR TITLE
REGD-143-summary3

### PR DESCRIPTION
**Pull request by mingjiecn**
What I did in this ticket:
1. register plugin zoomPlugin and Datalabels
2. update the way to use forwardRef for DnaLogo.
3. update the way to use Image RegulomeLogoFinal
4. some other minor changes

I am trying to deploy a demo but not working so far. Checking with Otto to see if I can deploy a demo.

### CODE REVIEW

File: components/regulome-version-tag.js [hunk 1]
```diff
@@ -13,8 +13,10 @@ export default function RegulomeVersionTag({ version = "2.2" }) {
         <Image
           src="/RegulomeLogoFinal.gif"
           alt="clickable image"
-          width="300"
-          height="96"
+          width="0"
+          height="0"
+          sizes="100vw"
```
Comment (forresttanaka) [inline components/regulome-version-tag.js:18]: Does `sizes` override the zero width and height? I couldn’t see any NextJS documentation either way.

File: components/tissue-score-bar.js [hunk 2]
```diff
@@ -18,8 +18,7 @@ ChartJS.register(
   BarElement,
   Title,
   Tooltip,
-  Legend,
-  Datalabels
+  Legend
```
Comment (forresttanaka) [inline components/tissue-score-bar.js:21]: I haven’t worked with chartjs in a long time, and never with react-chartjs-2, so I really don’t know for sure. But it looks like `ChartJS.register()` is intended just for global configurations. They describe an instance-based configration https://chartjs-plugin-datalabels.netlify.app/guide/getting-started.html#registration. I don’t know if that works for you, but I feel like the last `ChartJS.register` that happens to get called during a reload would get used globally. The last `ChartJS.register` called won’t necessarily be the one above the component it’s supposed to apply to.

**Review by forresttanaka — APPROVED** [review]
Approving, though one aspect makes me feel uneasy, and I describe that in the review comments.

File: components/regulome-version-tag.js (hunk 1 above)
Comment (mingjiecn) [reply components/regulome-version-tag.js:18]: I found this here: https://stackoverflow.com/questions/69230343/nextjs-image-component-with-fixed-witdth-and-auto-height.
It dose help to remove the warning.

Comment (forresttanaka) [reply components/regulome-version-tag.js:18]: Oh, I see. 👍

**Pull request merged**